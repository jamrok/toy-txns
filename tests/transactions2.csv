type,        client,  tx,  amount
deposit,     3,       1,   5.4321
deposit,     1,       1,   2.0
deposit,     2,       2,   2.0
deposit,     1,       3,   2.0
withdrawal,  1,       4,   3.5
withdrawal,  2,       5,   3.0
dispute,     3,       2,
withdrawal,  3,       2,   2.0
withdrawal,  1,       6,   0.1
dispute,     1,       1,
dispute,     1,       6,
chargeback,  1,       6,
resolve,     1,       6,
chargeback,  1,       1,
