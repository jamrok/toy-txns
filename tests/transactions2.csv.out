client,available,held,total,locked
1,0.4,0,0.4,true
2,2,0,2,false
3,3.4321,0,3.4321,false
